pages/common/git-summary.md

```diff
@@ -0,0 +1,17 @@
+# git summary
+
+> Display data about a Git repository, gives the following information: repository name, repository age, days active, total amount of commits, total amount of files and committer information.
```

Comment:
```suggestion
> Display information about a Git repository.
```
I think there's no need to list all the infos it gives. Also, the command output is very descriptive.

Reply:
I think the information should be in depth for the description. It gives all the info.

Reply:
The exact output of a command would be something I'd look for in a man page, but not in a tldr page.

Reply:
Yeah.  I don't like the long description either.  There's nothing wrong with the user figuring that out themselves.